README.md

```diff
@@ -224,6 +224,39 @@ If your repo has a different command to run for migrations, you can configure it
 `DeclareSchema.db_migrate_command = "bundle exec rails db:migrate_immediate"`
 ```
 
+## The `belongs_to` Relation and Indexes
+By default, `declare_schema` creates an index for `belongs_to` relations. If an index
+subsequently needs to be removed, it must be specifically updated in the relation statement.
+For example:
+
+```ruby
+belongs_to :other_table, null: false, index: false
+```
+
+The schema should then be updated, e.g.:
+
+```shell
+bundle exec rails g declare_schema:migration
+```
+
+## Ignored Tables
+`declare_schema` by default ignores these tables:
```

Comment:
I think an intro sentence would be good here as well. Something like
```
`declare_schema` can ignore any tables whose schema is managed elsewhere and not by the `declare_schema` gem.

Reply:
Section rewritten